### ISSUE TITLE
DYMO Label.app does not detect LabelWriter

### ISSUE THREAD

**Issue by lasar** [michaelrsweet/lprint#136]
Good evening,

I got lprint installed and running on Raspbian on a Raspberry Pi without a hitch. I was able to configure my Dymo LabelWriter 450 DUO for both Label and Tape printing and can print from my Mac to both.

However the original DYMO Label.app only deigns to work if it sees a Dymo printer, and apparently the lprint-based printers don't make the cut. I presume because it's not the original printer drivers.

They show up like this in my printer settings:
<img width="259" alt="image" src="https://github.com/michaelrsweet/lprint/assets/633673/09ef44f5-b58e-4331-9cd9-53a9fd52ed65">

I alternatively tried manually selecting the original driver when adding the printer on my mac. The printer then shows up with the original icon instead (honestly that's the only visible difference - the driver has the same name in the UI and I haven't dug deeper). Now I can run the Dymo app as expected. However any prints (from the dymo app or anywhere else) get accepted without error but nothing gets printed. I assume this is to be expected since it's not the driver that the lprint server expects.

Long story short: Is there a way to lull the Dymo app into believing these are "real" printers so it will do its job?

Thanks for all your hard work!

**Comment by michaelrsweet** [comment]
I don't think so - the DYMO application is hardcoded for local printers...

**Comment by lasar** [comment]
I'm positive that it works with networked Dymos as well, but apparently only when using the stock Dymo drivers. But if anything can be done about that I have no idea.

Anyway, thanks for checking in! I'll probably end up making my own little web interface for printing my usual labels so I can drop the Dymo app completely.
